scripts/translations/base_string_script.py

```diff
@@ -10,8 +10,8 @@
 
 class BaseStringScript:
 
-  def ProcessTag(self, lines):
-    return lines
+  def ProcessTag(self, line):
```

Comment:
Rename doesn't bother me but just FYI this is an array of multiple lines in the case where the `<string>` tag occupies more than one line in the XML file.